src/pharmpy/model/statements.py

```diff
@@ -908,24 +908,32 @@ def get_n_connected(self, comp):
     def dosing_compartment(self):
         """The dosing compartment
 
-        A dosing compartment is a compartment that receives an input dose. Only one dose
-        compartment is supported.
+        A dosing compartment is a compartment that receives an input dose. Multiple
+        dose compartments are supported.
 
         Returns
         -------
-        Compartment
-            Dosing compartment
+        tuple
+            A tuple of dose compartments
 
         Examples
         --------
         >>> from pharmpy.modeling import load_example_model
         >>> model = load_example_model("pheno")
-        >>> model.statements.ode_system.dosing_compartment
+        >>> model.statements.ode_system.dosing_compartment[0]
         Compartment(CENTRAL, amount=A_CENTRAL, dose=Bolus(AMT, admid=1))
         """
+        dosing_comps = tuple()
         for node in _comps(self._g):
             if node.dose is not None:
-                return node
+                if node.name != self.central_compartment.name:
+                    dosing_comps = tuple([node]) + dosing_comps
```

Comment:
How come it is `tuple([node]) + dosing_comps`, isn't it possible to do `(node,) + dosing_comps`?